modules/ROOT/pages/log-points.adoc

```diff
@@ -54,8 +172,6 @@ You can monitor the following request header log points: +
 * *Referrer* +
 You can click the blue box to the left of *Header*, to select all log points. +
 Select *Body* to monitor the *Request Body*.
```

Comment:
Reevaluate whether we really need to list all of these, given the image provided.